redbot/core/modlog.py

```diff
@@ -536,20 +536,20 @@ async def message_content(self, embed: bool = True):
             user = filter_mass_mentions(filter_urls(user))  # Further sanitization outside embeds
             case_text = ""
             case_text += "{}\n".format(title)
-            case_text += _("**User:** {}\n").format(user)
-            case_text += _("**Moderator:** {}\n").format(moderator)
+            case_text += f"{bold(_('User'))}: {user}\n"
+            case_text += f"{bold(_('Moderator'))}: {moderator}\n"
             case_text += "{}\n".format(reason)
             if until and duration:
-                case_text += _("**Until:** {}\n**Duration:** {}\n").format(until, duration)
+                case_text += f"{bold(_('Until'))}: {until}\n{bold(_('Duration'))}: {duration}\n"
             if self.channel:
                 if isinstance(self.channel, int):
-                    case_text += _("**Channel**: {} (Deleted)\n").format(self.channel)
+                    case_text += f"{bold(_('Channel'))}: {self.channel} (Deleted)\n"
                 else:
-                    case_text += _("**Channel**: {}\n").format(self.channel.name)
+                    case_text += f"{bold(_('Channel'))}: {self.channel.name}\n"
             if amended_by:
-                case_text += _("**Amended by:** {}\n").format(amended_by)
+                case_text += f"{bold(_('Amended by'))}: {amended_by}\n"
             if last_modified:
-                case_text += _("**Last modified at:** {}\n").format(last_modified)
+                case_text += f"{bold(_('Last modified at'))}: {last_modified}\n"
```

Comment:
There's some inconsistency in this PR of whether `:`s should go inside the translated string or outside of it, those should probably be done the same way each time.